### PR TITLE
npm-why: add page

### PR DESCRIPTION
**Pull request by sbrl**
Another day, another page on my continuing quest to have a page for every command in my shell history :P

### CODE REVIEW

**Comment by mebeim** [comment]
LOLl does a command like this even need a page? I mean, who installs this without knowing its one and only function? But well, I guess it's ok.

File: pages/common/npm-why.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# npm-why
+
+> Identifies why an npm package is installed.
+> Homepage: <https://www.npmjs.com/package/npm-why>
```
Comment (pxgamer) [inline pages/common/npm-why.md]:
```suggestion
> Homepage: <https://www.npmjs.com/package/npm-why>.
```

**Comment by sbrl** [comment]
Fixed - thanks @pxgamer!

@mebeim: You should check out the page for the `arch` command.

**Comment by mebeim** [comment]
@sbrl ok fair enough, haha. Gotta love such simple commands :rofl:

**Comment by sbrl** [comment]
Absolutely!

Check out #2213 for a list of commands we're currently working towards.

**Pull request merged**